### ISSUE TITLE
[Bug report]: 

### ISSUE THREAD

**Issue by Deep-Sea-888** [iotaledger/firefly#5427]
### Platform

Desktop

### Version

2.1.0-alpha-2

### Description

When clicking [find wallets], firefly asks for your password input. Error than is trigger. 

Error message is closed-out and then can run [find wallets] trigger.

### Expected behaviour

Have firefly run [find wallet] action without an error message.

### Actual behaviour

Receive error message below:

App Version: 2.1.0-alpha-2

platform: win32
platformVersion: 10.0.19044
platformArchitecture: x64
cpuCount: 16
totalMem: 28620.3 MB
freeMem: 13710.4 MB

Error Type: [Main Context] Unhandled Rejection

Error: Failed to open: The system cannot find the file specified. (0x2)

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. Click settings
2. Click [Advanced]
3. Click [Find Wallets]
4. Click [Search]
5. Firefly requests password.
6. Enter password and click [Unlock]

### Operating System

Windows

### Diagnostics

```shell
App Version: 2.1.0-alpha-2

platform: win32
platformVersion: 10.0.19044
platformArchitecture: x64
cpuCount: 16
totalMem: 28620.3 MB
freeMem: 13710.4 MB

Error Type: [Main Context] Unhandled Rejection

Error: Failed to open: The system cannot find the file specified. (0x2)
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

None

### Crash Status

None of the above

### Error Messages

App Version: 2.1.0-alpha-2

platform: win32
platformVersion: 10.0.19044
platformArchitecture: x64
cpuCount: 16
totalMem: 28620.3 MB
freeMem: 13710.4 MB

Error Type: [Main Context] Unhandled Rejection

Error: Failed to open: The system cannot find the file specified. (0x2)

### Error Logs

```shell
App Version: 2.1.0-alpha-2

platform: win32
platformVersion: 10.0.19044
platformArchitecture: x64
cpuCount: 16
totalMem: 28620.3 MB
freeMem: 13710.4 MB

Error Type: [Main Context] Unhandled Rejection

Error: Failed to open: The system cannot find the file specified. (0x2)
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MarkNerdi** [comment]
@Deep-Sea-888 Did you update your Firefly to 2.1.0? If yes, does this error still occur?

**Comment by Deep-Sea-888** [comment]
Yes I did and this error has not occured once!

**Comment by MarkNerdi** [comment]
Fixed in 2.1.0
